packages/cli/tests/e2e/bot/ProvisionResourceProjectWithNewBot.tests.ts

```diff
@@ -67,9 +71,13 @@ describe("Provision", function () {
       // Get context
       const context = await readContextMultiEnv(projectPath, env);
 
-      // Validate Aad App
-      const aad = AadValidator.init(context, false, AppStudioLogin);
-      await AadValidator.validate(aad);
+      // Only validate aad when preview flag is turned off
+      // since no sso enabled for bot under preview mode.
+      if (!isPreviewFeaturesEnabled()) {
```

Comment:
What's this AAD for? If it is the Bot AAD, it doesn't matter whether preview flag is on or off.

Reply:
It's not Bot AAD, instead it means AAD Plugin.